test/workers/tipline_newsletter_worker_test.rb

```diff
@@ -18,44 +18,60 @@ def setup
   end
 
   test "should not crash if error happens when sending newsletter to some subscriber" do
-    Bot::Smooch.stubs(:send_final_messages_to_user).raises(StandardError)
+    Bot::Smooch.stubs(:send_message_to_user).raises(StandardError)
     assert_nothing_raised do
       TiplineNewsletterWorker.perform_async(@team.id, 'en')
     end
-    Bot::Smooch.unstub(:send_final_messages_to_user)
+    Bot::Smooch.unstub(:send_message_to_user)
```

Comment:
**info** we shouldn't need this line anymore since we're tearing down stubs at the end of every test. I've just been ripping out the lines in tests when I touch the test or file

Reply:
Yes! Sometimes I still forget hehe